winlogbeat/sys/event.go

```diff
@@ -133,6 +133,7 @@ func (t *TimeCreated) UnmarshalXML(d *xml.Decoder, start xml.StartElement) error
 // EventData contains the event data. The EventData section is used if the
 // message provider template does not contain a UserData section.
 type EventData struct {
+	// TargetUserName    string `xml:"TargetUserName,attr"`
```

Comment:
Not familiar with building these structs backed by a document. That's what I tried.